docs/mivot/index.rst

```diff
@@ -164,6 +164,27 @@ with the `astropy.io.votable` API:
 
 In this case, it is up to the user to ensure that the read data rows are those mapped by the Mivot annotations.
 
+Get a SkyCoord Instance Directly From the Annotations
+-----------------------------------------------------
+
+Once you get a ``MivotInstance`` representing the last row read, you can use it to create an ``astropy.SkyCoord`` object.
+
+.. code-block:: python
+    :caption: Accessing the model view of Astropy table rows
+
+	from pyvo.mivot import MivotViewer
+    from pyvo.mivot.utils.dict_utils import DictUtils
```

Comment:
why do you import DictUtils?

Reply:
This was a trace of removed statement: corrected